nvm.sh

```diff
@@ -1186,6 +1186,7 @@ nvm() {
       echo "  nvm help                              Show this message"
       echo "  nvm --version                         Print out the latest released version of nvm"
       echo "  nvm install [-s] <version>            Download and install a <version>, [-s] from source. Uses .nvmrc if available"
+      echo "    --reinstall-packages-from=<version> When installing, reinstall packages installed in <node|iojs|node version number>"
```

Comment:
`node|iojs|node version number` isn't entirely accurate, but perhaps (in a separate PR) we should add a line or two above "Usage" explaining what a "version-ish" is (like a commit-ish in git), and then use that term in the command docs whenever it applies.

I'm not attached to the name "version-ish" so other suggestions would be welcome.